### DISCUSSION TITLE
Using the plugin without providing a password

### DISCUSSION THREAD

**Discussion by AlexGustafsson** [AlexGustafsson/homebridge-wol#155] [Questions] [answered]
_Note: moved from a comment on an issue authored by @flocorbab._

Original comment:
> bonjour,
j'aurais une petite question sur le plugin Homebridge-Wol,
je l'ai installé tout fonctionne mais aujourd'hui j'ai décidé d'enlever
le mot de passe de mon pc, alors j'ai modifié le fichier config
d'homebridge en enlevant le mot de passe après " %" mais aprés redemarrage j'ai essayé d'éteindre le pc avec homebridge sans
succès
"'The username or password was not correct.\n' +"
Auriez vous une solution pour utiliser ce plugin sans mot de passe?
Merci de votre aide,
passez une bonne journée,
Florian.

Translation (using Google translate):
> Hello,
I have a quick question about the Homebridge-Wol plugin,
I installed it everything works but today I decided to remove
my pc's password, so i modified the config file
homebridge by removing the password after "%" but after restarting I tried to shut down the pc with homebridge without
success
"'The username or password was not correct. \ N' +"
Do you have a solution to use this plugin without password?
> Thank you for your help,
have a nice day,
Florian.

**Comment by AlexGustafsson** [comment]
This plugin provides no commands by default, it allows for _any_ command to be used. As such there are infinite possibilities of commands to run. It's not clear from your question what command you're having an issue with. Judging from the use of '%' I assume that you're using a command like the following: 

```sh
net rpc shutdown --ipaddress 192.168.1.151 --user username%password
```

To use the same command without a password, try to exchange it with the following:

```sh
net rpc shutdown --ipaddress 192.168.1.151 --user username
```

I haven't tried it out myself, rather googled for the solution and found this page: https://ubuntuforums.org/archive/index.php/t-173440.html.

If that doesn't work for you, try to search for it yourself using the queries `use net rpc shutdown without a password`, `samba shutdown user without password` etc.

Best of luck.

Translation (using Google translate):

> Ce plugin ne fournit aucune commande par défaut, il permet l'utilisation de _any_ commande. En tant que tel, il existe des possibilités infinies de commandes à exécuter. Il n'est pas clair d'après votre question avec quelle commande vous rencontrez un problème. À en juger par l'utilisation de '%', je suppose que vous utilisez une commande comme celle-ci:
```sh
net rpc shutdown --ipaddress 192.168.1.151 --user username%password
```
> Pour utiliser la même commande sans mot de passe, essayez de l'échanger avec ce qui suit:
```sh
net rpc shutdown --ipaddress 192.168.1.151 --user username
```
> Je ne l'ai pas essayé moi-même, j'ai plutôt cherché la solution sur Google et j'ai trouvé cette page: https://ubuntuforums.org/archive/index.php/t-173440.html.
> Si cela ne fonctionne pas pour vous, essayez de le rechercher vous-même en utilisant les requêtes `utiliser net rpc shutdown sans mot de passe`,` samba shutdown user without password` etc.
> Bonne chance.

**Comment by flocorbab** [comment, marked as answer]
thank you for your comeback,I did not find the solution, so I added a password to the pc and subsequently removed the password request at bootDe : Alex GustafssonEnvoyé le :dimanche 11 avril 2021 13:43À : AlexGustafsson/homebridge-wolCc : flocorbab; MentionObjet :Re: [AlexGustafsson/homebridge-wol] Using the plugin without providing a password (#155) This plugin provides no commands by default, it allows for any command to be used. As such there are infinite possibilities of commands to run. It's not clear from your question what command you're having an issue with. Judging from the use of '%' I assume that you're using a command like the following:net rpc shutdown --ipaddress 192.168.1.151 --user username%passwordTo use the same command without a password, try to exchange it with the following:net rpc shutdown --ipaddress 192.168.1.151 --user usernameI haven't tried it out myself, rather googled for the solution and found this page: https://ubuntuforums.org/archive/index.php/t-173440.html.If that doesn't work for you, try to search for it yourself using the queries use net rpc shutdown without a password, samba shutdown user without password etc.Best of luck.Translation (using Google translate):Ce plugin ne fournit aucune commande par défaut, il permet l'utilisation de any commande. En tant que tel, il existe des possibilités infinies de commandes à exécuter. Il n'est pas clair d'après votre question avec quelle commande vous rencontrez un problème. À en juger par l'utilisation de '%', je suppose que vous utilisez une commande comme celle-ci:net rpc shutdown --ipaddress 192.168.1.151 --user username%passwordPour utiliser la même commande sans mot de passe, essayez de l'échanger avec ce qui suit:net rpc shutdown --ipaddress 192.168.1.151 --user usernameJe ne l'ai pas essayé moi-même, j'ai plutôt cherché la solution sur Google et j'ai trouvé cette page: https://ubuntuforums.org/archive/index.php/t-173440.html.Si cela ne fonctionne pas pour vous, essayez de le rechercher vous-même en utilisant les requêtes utiliser net rpc shutdown sans mot de passe, samba shutdown user without password etc.Bonne chance.—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.  
	
      
		Garanti sans virus. www.avast.com
